benchmark.py

```diff
@@ -81,10 +84,12 @@
 
 tictoc = lambda: (use_cuda and torch.cuda.synchronize()) or time.time()
 
-batch_shape = [args.B, args.T * args.sample_rate
+batch_shape = [args.B, int(args.T * args.sample_rate)
 				] if args.frontend else [args.B, args.num_input_features, int(args.T / args.window_stride)]
-if batch_shape[-1] % 128 != 0:
+
+if args.satisfy_t_shape_divisibility_by_128 and batch_shape[-1] % 128 != 0:
```

Comment:
please remove constant duplication. instead of `satisfy_t_shape_divisibility_by_128 `, please make it an `int` argument `input_time_dim_multiple` - just like you did elsewhere